cmd/config/set.go

```diff
@@ -273,3 +326,13 @@ func expandHomePath(file string) string {
 	}
 	return file
 }
+
+func clearContext(ctxPtr *Context) {
```

Comment:
consider renaming to clearAuthFields()
In the future, the context may contain non-auth-related values (e.g., `devenv` or `uql.apiver`). What we want to wipe are only the authentication fields that are likely no longer applicable.